tests/controls/listboxtest.cpp

```diff
@@ -109,12 +109,21 @@ void ListBoxTestCase::Sort()
 
     m_list->Append(testitems);
 
+#ifndef __WXQT__
```

Comment:
The different sort order should ideally be documented (I assume we can't change it).

Reply:
we can change the default sort order by overriding the  `operator <` :
```
    virtual bool wxQtListWidgetItem::operator<(const QListWidgetItem& other) const override
    {
        const auto str1 = text();
        const auto str2 = other.text();
        return str1.compare(str2, Qt::CaseInsensitive) < 0;
    }
```
Sure it will make the test happy then (as it passes successfully with it) but how about the end user ? would the list be sorted the way he/she wants/expects ? because the above code doesn't take into account the (locale-specific) rules on how to compare  strings!
IMHO `wxQt` and `wxGTK` (before this commit https://github.com/wxWidgets/wxWidgets/commit/625ed743015f2da84db47a6851e6011bd08f4bd2) looks more correct to me because they respect the `LC_COLLATE` setting of the location.